build/charts/flow-aggregator/conf/flow-aggregator.conf

```
@@ -72,9 +72,23 @@ clickHouse:
   # Database is the name of database where Antrea "flows" table is created.
   database: "default"
 
-  # DatabaseURL is the url to the database. TCP protocol is required.
+  # DatabaseURL is the url to the database. TCP/HTTP/HTTPS protocols are supported.
```

Why don't we remove the `tls: enable` flag below and add TLS here?
```suggestion
  # Provide the database URL as string with format <Protocol>://<ClickHouse server FQDN or IP>:<ClickHouse port>. The protocol has to be one from below: "tcp", "tls", "http", "https"
```